### PR TITLE
Validate message ID for CFR experiment

### LINKED ISSUE #159: Ensure the message naming convention for the CFR experiments
This allows Jetstream to perform the automated analysis, which requires messages of CFR experiments to use the branch slug as the message ID.

### PR DESCRIPTION
**Pull request by ncloudioj**
This fixes #159.

@piatra Yet another tape on the duct :P.

Note that this only enforces the message_id for CFR experiments. I've also deleted the recipe for Moments page so that we can clean up the validation script a bit.

### CODE REVIEW

File: scripts/validate.py [hunk 1]
```diff
@@ -192,10 +164,25 @@ def get_branch_message(branch):
...
+    types such as CFR in order to support the automated analysis feature of
+    Jetstream.
+    """
+    message_type, branch_message = get_branch_message(branch)
+    if branch_message is None:
+        return
+
+    if message_type == "cfr":
+        print(f"\tValidate experiment message ID for branch {branch['slug']}")
+        assert branch_message.id == f"{exp_slug}:{branch['slug']}", \
```
Comment (piatra) [inline scripts/validate.py:178]: Do we also have to do this for `bucket_id` is this change different from https://bugzilla.mozilla.org/show_bug.cgi?id=1649642 ?

Comment (ncloudioj) [reply scripts/validate.py:178]: You're right, we do collect both for experiment users.

**Review by piatra — APPROVED** [review]
This looks good thanks. Can you please update the dummy experiment to be a `cfr` experiment (doesn't need real content) just enough so we run the id/bucket_id validation on it.

**Comment by ncloudioj** [comment]
> This looks good thanks. Can you please update the dummy experiment to be a `cfr` experiment (doesn't need real content) just enough so we run the id/bucket_id validation on it.

Yeah, will do. And that's a good idea!

**Pull request merged**